### PR TITLE
fix(jaas) support for destroying & reinitializing the react native bridge

### PR DESCRIPTION
**Pull request by alexbumbu**

### CODE REVIEW

File: ios/sdk/src/JitsiMeet.m [hunk 1]
```diff
@@ -119,6 +123,14 @@ - (BOOL)application:(UIApplication *)app
 
 #pragma mark - Utility methods
 
+- (void)instantiateReactNativeBridge {
+    _bridgeWrapper = [[RCTBridgeWrapper alloc] init];
```
Comment (alexbumbu) [inline ios/sdk/src/JitsiMeet.m:127]: maybe I should add a check to skip if already instantiated

Comment (saghul) [reply ios/sdk/src/JitsiMeet.m:127]: Yeah let's do that please.

**Comment by saghul** [comment]
LGTM modulo what you already pointed out yourself.

File: ios/sdk/src/JitsiMeet.m [hunk 2]
```diff
@@ -214,12 +230,20 @@ - (NSDictionary *)getDefaultProps {
     return _defaultConferenceOptions == nil ? @{} : [_defaultConferenceOptions asProps];
 }
 
+- (RCTBridgeWrapper *)bridgeWrapper {
+    if (_bridgeWrapper == NULL) {
+        _bridgeWrapper = [[RCTBridgeWrapper alloc] init];
```
Comment (saghul) [inline ios/sdk/src/JitsiMeet.m:235]: This feels wrong, we shouldn't be creating one here. What are we trying to solve with this?

File: ios/sdk/src/JitsiMeet.m [hunk 3]
```diff
@@ -50,7 +54,7 @@ + (instancetype)sharedInstance {
 
 - (instancetype)init {
     if (self = [super init]) {
-        // Initialize the on and only bridge for interfacing with React Native.
+        // Initialize the one and only bridge for interfacing with React Native.
         _bridgeWrapper = [[RCTBridgeWrapper alloc] init];
```
Comment (saghul) [inline ios/sdk/src/JitsiMeet.m:58]: We should only create the bridge if it doesn't exist yet, since an application could have instantiated it early now.

**Review by saghul — COMMENTED** [review]
A couple of comments, PTAL.

File: ios/sdk/src/JitsiMeet.m (hunk 2 above)
Comment (alexbumbu) [reply ios/sdk/src/JitsiMeet.m:235]: this is how lazy initialization is done in obj c

File: ios/sdk/src/JitsiMeet.m (hunk 3 above)
Comment (alexbumbu) [reply ios/sdk/src/JitsiMeet.m:58]: since JitsiMeet is a singleton will be called only once

**Review by saghul — APPROVED** [review]

**Pull request merged**